nilearn/reporting/sphinx_report.py

```diff
@@ -1,44 +1,48 @@
 # Scraper for sphinx-gallery
-# Copied from https://github.com/mne-tools/mne-python/
+# Inspired from https://github.com/mne-tools/mne-python/
+import weakref
 
-import os.path as op
-from shutil import copyfile
-from nilearn.reporting import HTMLReport
+from nilearn.reporting import HTMLDocument
 
 
 _SCRAPER_TEXT = '''
 .. only:: builder_html
 
-    .. container:: row
+    .. container:: row sg-report
 
         .. raw:: html
 
-            <iframe class="sg_report" frameBorder="0" width="None" height="None"
-                srcdoc="{0}"></iframe>
+            {0}
 
 '''  # noqa: E501
 
+def indent_and_espace(text, amount=12):
+    "Indent, skip empty lines, and escape string delimiters"
+    return (''.join(amount * ' ' + line.replace("'", '"')
+                for line in text.splitlines(True)
+                if line)
+            + 12 * ' ')
```

Comment:
Is there a reason this is not as above ?

```suggestion
            + amount * ' ')
```